yt_dlp/YoutubeDL.py

```diff
@@ -1402,7 +1402,7 @@ def check_filter():
             return None
 
         if self.in_download_archive(info_dict):
-            reason = '%s has already been recorded in the archive' % video_title
+            reason = f'{self._format_screen(video_title, self.Styles.FILENAME)} has already been recorded in the archive'
```

Comment:
Maybe not, but I would like to see both `id` and `title` if available. I'd like to know @pukkandan's opinion on this matter.

Reply:
Sorry, I did not get notifications for the added labels.
@pukkandan I was waiting for your reply to the above comment before making any more changes

> I'd like to know @pukkandan's opinion on this matter.

Reply:
I agree that it makes sense to show both